sound/soc/sof/intel/byt.c

```diff
@@ -145,33 +145,33 @@ static void byt_dump(struct snd_sof_dev *sdev, u32 flags)
 	struct sof_ipc_dsp_oops_xtensa xoops;
 	struct sof_ipc_panic_info panic_info;
 	u32 stack[BYT_STACK_DUMP_SIZE];
-	u32 status, panic, imrd, imrx;
+	u64 status, panic, imrd, imrx;
```

Comment:
yes, but you should add in the commit message that this was a copy/paste mistake. The changes were initially done for BDW which does have 32-bit registers

Also please add a Fixes: tag, so that I don't have to do this before I submit upstream

Reply:
> yes, but you should add in the commit message that this was a copy/paste mistake. The changes were initially done for BDW which does have 32-bit registers
> 
> Also please add a Fixes: tag, so that I don't have to do this before I submit upstream

Sure, doing it now.

Reply:
BTW, @plbossart usually how do you add this 'Fixes:' tag, amend it manually?

Reply:
yes, git commit --amend --signoff

Reply:
wrong Fixes tag

````
9e42c5ca4a276 (Liam Girdwood         2019-04-12 11:08:44 -0500 143) static void byt_dump(struct snd_sof_dev *sdev, u32 flags)
9e42c5ca4a276 (Liam Girdwood         2019-04-12 11:08:44 -0500 144) {
9e42c5ca4a276 (Liam Girdwood         2019-04-12 11:08:44 -0500 145)     struct sof_ipc_dsp_oops_xtensa xoops;
9e42c5ca4a276 (Liam Girdwood         2019-04-12 11:08:44 -0500 146)     struct sof_ipc_panic_info panic_info;
9e42c5ca4a276 (Liam Girdwood         2019-04-12 11:08:44 -0500 147)     u32 stack[BYT_STACK_DUMP_SIZE];
3a9e204d4e369 (Liam Girdwood         2019-09-27 15:05:37 -0500 148)     u32 status, panic, imrd, imrx;
````

and wrong format. Use checkpatch....

Reply:
Oh, this is error-prone by doing it manually.

Reply:
@keyonjie can you please fix the commit message. I gave you all the information... Thanks!